bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml

```diff
@@ -367,6 +367,8 @@ spec:
                   capabilities:
                     drop:
                     - ALL
+                  seccompProfile:
```

Comment:
This should not be readded. It was removed to be able to support an older OpenShift.

Reply:
so there is seccompProfile error  when I tested deploying bundle in local also OLM test is not stable. In OLM test deployment status show seccompProfile error so it doesn't create pod. actually it is randomly happens so I need a help